xbmc/music/MusicDatabase.cpp

```diff
@@ -8716,6 +8956,13 @@ bool CMusicDatabase::IsSongAlbumArtist(int idSong, int idArtist)
   return GetSingleValue(strSQL).empty();
 }
 
+bool CMusicDatabase::IsAlbumBoxset(int idAlbum)
+{
+  std::string strSQL = PrepareSQL("SELECT bBoxedSet FROM album WHERE idAlbum = %i", idAlbum);
+  int isBoxSet = strtol(GetSingleValue(strSQL).c_str(), NULL, 10);
```

Comment:
To be honest, I don't get any compiler warnings other than some right at the beginning to do with swig and a few to do with ffmpeg.  Other than those, the compile is clean on my machine but I can change it anyway if that makes it more compatible with long term goals.  Also, with that in mind, I have added Clang to my IDE and run it against all my code, so you might see the odd line or two in the future where formatting has changed slightly because of that.  However, I'm doing that to try and avoid any such issues in the future as I know of at least one team member that is very keen on clang compliance ;)

Reply:
Yes, I was going to save that formatting conformance "delight" for the end :)
You could just run clang-format-diff.py on the final PR. Doing as you go do try to ensure it only formats the things you change (the integrated version of Clang went wild on my IDE on one occassion).